components/feature/search/src/main/java/mozilla/components/feature/search/SearchUseCases.kt

```diff
@@ -37,8 +38,8 @@ class SearchUseCases(
         /**
          * Triggers a search in the currently selected session.
          */
-        override fun invoke(searchTerms: String) {
-            invoke(searchTerms, sessionManager.selectedSession)
+        override fun invoke(searchTerms: String, searchEngine: SearchEngine?) {
+            invoke(searchTerms, sessionManager.selectedSession, searchEngine)
         }
```

Comment:
I wonder if we should add the `searchEngine` parameter to the constructor instead of the `invoke` method: Other components will call `invoke` and not provide the parameter (because they have no knowledge about the default). So instead the app needs to configure the use case to use the right search engine when something calls `invoke` on it.